f8a_report/graph_report_generator.py

```diff
@@ -86,6 +86,38 @@ def generate_report_for_unknown_epvs(epv_list):
     return report_result
 
 
+def find_ingested_epv(ecosystem, pvlist):
```

Comment:
Actually you dont need this function. You can instead use the function generate_report_for_unknown_epvs. You can pass a list of EPVs and it will return back a list with the key as eco@DELIM@pkg@DELIM@ver and the value as "false" if its unknown and "true" if its known

Reply:
@yzainee aware of that fact. Although the nature of those two functions are almost the same, the input that I can pass and the output that I need from that function are somewhat different. This new function allows the implementation of the method `UnknownDepsReportHelper.get_current_ingestion_status` very simple.